Documentation/gettingstarted/k8s-install-kubespray.rst

```diff
@@ -141,15 +141,24 @@ Installing Kubernetes cluster with Cilium as CNI
 
 Kubespray uses Ansible as its substrate for provisioning and orchestration. Once the infrastructure is created, you can run the Ansible playbook to install Kubernetes and all the required dependencies. Execute the below command in the kubespray clone repo, providing the correct path of the AWS EC2 ssh private key in ``ansible_ssh_private_key_file=<path to EC2 SSH private key file>``
 
-We recommend using the `latest released Cilium version`_ by editing ``roles/download/defaults/main.yml``. Open the file, search for ``cilium_version``, and replace the version with the latest released. As an example, the updated version entry will look like: ``cilium_version: "v1.2.0"``.
-
+We recommend using the `latest released Cilium version`_ by passing the variable when running the ``ansible-playbook`` command.
+For example, you could add the following flag to the commend below: ``-e cilium_version=v1.11.0``.
```

Comment:
Thanks for the updates. I'm sorry, I missed this one on my first pass:
```suggestion
For example, you could add the following flag to the command below: ``-e cilium_version=v1.11.0``.
```

Reply:
😅 Thanks for all the feedback. We will make sure to triple-check the next time! 😄